src/phases/scale/MixPhases.yml

```diff
@@ -0,0 +1,49 @@
+PhaseSchemaVersion: 2018-07-01
+
+dbname: &dbname mix
+runtime: &runtime 7 minutes
+DocumentCount: &NumDocs 100000
+Filter: &filter {id: {^RandomInt: {min: 0, max: *NumDocs}}}
+string: &string {^FastRandomString: {length: 50}}
```

Comment:
This may be overkill, but the quality of random strings isn't important for this workload (I don't think), and there are a lot of them.